src/DDLQueryBuilder.php

```diff
@@ -167,27 +168,23 @@ public function renameColumn(string $table, string $oldName, string $newName): s
      * @param string $table The table name.
      * @param string $column The column name.
      *
-     * @throws Throwable In case when table doesn't contain a column.
-     *
-     * @return string The column definition.
+     * @return string The column definition or empty string in case when schema does not contain the table
+     * or the table doesn't contain the column.
      */
-    public function getColumnDefinition(string $table, string $column): string
+    private function getColumnDefinition(string $table, string $column): string
```

Comment:
```suggestion
    private function getColumnDefinition(string $table, string $column): ?string
```

Since the method is private now, it can change result type?
`null` in PHP means "no value" (no column definition).

Reply:
In some cases this will require to convert `null` to empty string.

https://github.com/yiisoft/db-mysql/pull/320/files#diff-405e82bbc7b84d42228151fdab40f37e331b0ee13a9af9975292bfd8e20b1fa1R40